roles/nos-crossposting-service/templates/nginx.conf.tpl

```diff
@@ -0,0 +1,39 @@
+server {
+        server_name {{ domain }};
+
+
+        location / {
+                proxy_pass         http://127.0.0.1:{{ crossposting_listen_address }}/;
+                proxy_redirect     off;
+
+                proxy_set_header Upgrade $http_upgrade;
+                proxy_set_header Connection "Upgrade";
+
+                proxy_set_header   Host             $host;
+                proxy_set_header   X-Real-IP        $remote_addr;
+                proxy_set_header   X-Forwarded-For  $proxy_add_x_forwarded_for;
+        }
+        location /metrics {
+                proxy_pass         http://127.0.0.1:{{ crossposting_listen_address }}/metrics;
```

Comment:
```suggestion
                proxy_pass         http://127.0.0.1:{{ crossposting_metrics_listen_address }}/metrics;
```

Reply:
I won't be creating separate addresses like this in the future since it makes no sense really with our setup.